core/connection/manager.go

```diff
@@ -117,7 +133,7 @@ type connectionManager struct {
 	eventPublisher           eventbus.Publisher
 	connectivityStatusSender connectivity.StatusSender
 	ipResolver               ip.Resolver
-	ipCheckParams            IPCheckParams
+	params                   Params
```

Comment:
`params` - too generic. Are you trying to introduce manager configuration here?

Reply:
Yes, this is connection manager Params. As it lives in connection package I didn't named it ConnectionnParams. The main purpose for now is to set different values for unit tests, but params can be parsed from flags if needed too.